src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/ComCase.kt

```diff
@@ -0,0 +1,68 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceKind
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceType
+import java.time.LocalDate
+
+enum class LicenceCreationType {
+  LICENCE_CHANGES_NOT_APPROVED_IN_TIME,
+  PRISON_WILL_CREATE_THIS_LICENCE,
+  LICENCE_CREATED_BY_PRISON,
+  LICENCE_NOT_STARTED,
+  LICENCE_IN_PROGRESS,
+}
+
+@Schema(description = "Describes an COM case")
+data class ComCase(
+  @Schema(description = "The full name of the person on licence", example = "John Doe")
+  val name: String?,
+
+  @Schema(description = "The case reference number (CRN) for the person on this licence", example = "X12444")
+  val crnNumber: String?,
+
+  @Schema(description = "The prison identifier for the person on this licence", example = "A9999AA")
+  val prisonerNumber: String?,
+
+  @Schema(description = "The date on which the prisoner leaves custody", example = "30/11/2022")
+  @JsonFormat(pattern = "dd/MM/yyyy")
+  val releaseDate: LocalDate?,
+
+  @Schema(description = "The date on which a list of returned COM cases has been sorted", example = "30/11/2022")
+  @JsonFormat(pattern = "dd/MM/yyyy")
+  val sortDate: LocalDate? = null,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏For the Approver caseload, we removed this and sorted based on release date from the backend. Could this be done here or is `sortDate` used for something else?

Reply:
the sort is done on the backend, I've removed sortDate from here and added to the frontend as it is just used to populate an attribute on the table so say what it is sorted by.